src/ft2font.h

```diff
@@ -91,6 +92,7 @@ class FT2Font
                                  FT_Int32 flags,
                                  FT_Error &charcode_error,
                                  FT_Error &glyph_error,
+                                 std::set<FT_String*> &seen_fonts,
```

Comment:
```suggestion
                                 std::set<FT_String*> &glyph_seen_fonts,
```

the compiler does not care, but people will.